config/profiles/kentik_snmp/cisco/cisco-all-devices.yml

```diff
@@ -79,6 +79,7 @@ metrics:
         name: cpmCPUMemoryFree
       - OID: 1.3.6.1.4.1.9.9.109.1.1.1.1.7
         name: cpmCPUTotal1minRev
+        tag: CPU
```

Comment:
Will this override the CPU and MemoryUtilization in the Cisco Nexus profile?

Reply:
Great call out! What do you think about https://github.com/kentik/ktranslate/pull/63/files#diff-691ad03eaf661c8effe86119ae7cbe127095b1a18f0603c9a7d7bde018030822R522

This checks for duplicate tags and keeps the longest oid. Testing Nexus, this successfully keeps the nexus specific oid while dropping the general one.

Reply:
elegant and simple; LGTM! 🚀